osu.Game/Overlays/Rankings/Displays/PerformanceDisplay.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Online.API.Requests;
+using osu.Game.Overlays.Rankings.Tables;
+using osu.Game.Users;
+using osu.Game.Online.API;
+
+namespace osu.Game.Overlays.Rankings.Displays
+{
+    public class PerformanceDisplay : RankingsDisplay
+    {
+        public readonly BindableWithCurrent<Country> Country = new BindableWithCurrent<Country>();
+        private readonly Bindable<RankingsSortCriteria> sort = new Bindable<RankingsSortCriteria>();
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            Country.BindValueChanged(_ => FetchRankings());
+            sort.BindValueChanged(_ => FetchRankings(), true);
+        }
+
+        protected override APIRequest CreateRequest() => new GetUserRankingsRequest(Current.Value, UserRankingsType.Performance, sort.Value, 1, Country.Value?.FlagName);
+
+        protected override Drawable CreateHeader() => new Container
+        {
+            RelativeSizeAxes = Axes.X,
+            AutoSizeAxes = Axes.Y,
+            Children = new Drawable[]
+            {
+                new FillFlowContainer
+                {
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Direction = FillDirection.Vertical,
+                    Children = new Drawable[]
+                    {
+                        new CountryFilter
+                        {
+                            Current = Country
+                        },
```

Comment:
has the same awkward spacing as the spotlights ranking display used to have

![2020-07-18-230529_1247x166_scrot](https://user-images.githubusercontent.com/20418176/87861924-407ebf80-c94b-11ea-9350-fed6a3cd31f0.png)

Reply:
It will be changed anyway in the near future (in line with the web)